contribute.md

```diff
@@ -5,16 +5,20 @@ title: Contributing to Foreman
 
 # Contributing
 
-The Foreman is an open-source project that's licensed under the GNU Public License version 3. Contributions of all types are gladly accepted!
+The Foreman is an open-source project that's licensed under the GNU Public License version 3.<br>
+Contributions of all types are gladly accepted!
+
 
 ## Code-related contributions
+**Follow the [handbook](/handbook.html) when contributing code to Foreman**
```

Comment:
I think this should get it's own `##` section so it shows up on the menu at the left. It's important enough to highlight it there.